pkg/controller/clusterinstallation/mattermost_test.go

```diff
@@ -47,10 +53,19 @@ func TestCheckMattermost(t *testing.T) {
 		},
 	}
 
+	fakeExecutor := newFakePodExecutor("5.28.0", nil)
+
 	apis.AddToScheme(scheme.Scheme)
 	s := scheme.Scheme
 	s.AddKnownTypes(mattermostv1alpha1.SchemeGroupVersion, ci)
-	r := &ReconcileClusterInstallation{client: fake.NewFakeClient(), scheme: s}
+
+	testServer, _, _ := testServerEnv(t, 200)
+	defer testServer.Close()
+
+	config := restConfig(testServer)
+	//cs := kFake.NewSimpleClientset()
```

Comment:
I think this can be also cleaned up, as the `PodExecutor` is mocked and we do not call anything actually.